parameter/ParameterMgr.cpp

```diff
@@ -924,15 +924,15 @@ CParameterMgr::CCommandHandler::CommandStatus CParameterMgr::statusCommandProces
 }
 
 /// Tuning Mode
-CParameterMgr::CCommandHandler::CommandStatus CParameterMgr::setTuningModeCommandProcess(const IRemoteCommand& remoteCommand, string& strResult)
+CParameterMgr::CCommandHandler::CommandStatus CParameterMgr::setTuningModeCommandProcess(const CRequestMessage& remoteCommand, string& strResult)
 {
-    if (remoteCommand.getArgument(0) == "on") {
+    if (remoteCommand.getArguments()[0] == "on") {
```

Comment:
Proposal: add an overloaded version of `CRequestMessage::getArguments`:

    const std::string &getArguments(std::vector::size_type index) const { return _arguments[index]; }